build.gradle

```diff
@@ -7,9 +7,12 @@ buildscript {
         classpath "org.shipkit:shipkit-changelog:1.2.0"
         classpath "org.shipkit:shipkit-auto-version:1.2.2"
         classpath "io.github.gradle-nexus:publish-plugin:1.0.0"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.9.20"
+        classpath "org.jetbrains.dokka:dokka-gradle-plugin:1.9.10"
```

Comment:
11. Centralize `buildscript` classpath, use the same plugin versions everywhere in the project. There's only one module, so the difference is not too visible, however now there's only one place plugins are defined, other than the sample/tests project.

This is not a necessity of this refactor, but a good practice regardless. The plugin classpath is defined here, while each module/project will define what plugins are **applied** from that classpath separately. (`apply plugin:` or `plugins { id...`)

Reply:
https://github.com/mockito/mockito-kotlin/pull/503 is a follow-up to this.